serialapi/controller_timeouts.go

```diff
@@ -0,0 +1,51 @@
+package serialapi
+
+import (
+	"errors"
+	"github.com/gozwave/gozw/frame"
+	"github.com/gozwave/gozw/protocol"
+	"github.com/gozwave/gozw/session"
+)
+
+type ControllerTimeouts struct {
+	AckTimeout  byte
+	ByteTimeout byte
+}
+
+const (
+	ackTimeout  = 0x96 // 0x95 = 150 -- multiply by 10 to get 1500ms
```

Comment:
Could you put a comment in here with a link to the documentation on this? (Even if it's just the section header)